openmrs/apps/clinical/app.json

```diff
@@ -36,34 +36,65 @@
             ]
         },
     	"diagnosisStatus": "Inactive",
-            "program": {
-                "patientInformation": {
-                    "ageLimit": 100
-                },
-                "ID_Number": {
-                    "required" : true
-                },
-		        "PROGRAM_MANAGEMENT_PATIENT_COMORBIDITES": {
-                    "excludeFrom": ["TB_PROGRAM_KEY"]
-                },
-                "PROGRAM_MANAGEMENT_LABEL_THERAPEUTIC_LINE": {
-                    "excludeFrom": ["TB_PROGRAM_KEY"]
-                }
-            },
-            "patientContext": {
-        		"programAttributes": [
-            		"Id_Number",
-            		"Doctor",
-            		"Stage"
-        		],
-        		"personAttributes": [
-            		"class",
-            		"caste"
-        		],
-        		"additionalPatientIdentifiers": [
-           			"National ID"
+        "program": {
+            "patientInformation": {
+                "ageLimit": 100
+            },
+            "PROGRAM_MANAGEMENT_PATIENT_COMORBIDITES": {
```

Comment:
@lauravignoli Do you know if it is a requirement to maintain the field order as in the Excel sheet?

Reply:
I don't think so

Reply:
I think let's leave it for now?